### PR TITLE
star: add 2.7.10

### PR DESCRIPTION
**Pull request by A-N-Other**
Adding checksums for the `star@2.7.10` versions. No other changes required to get this installed.

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
Confirmed the two version sha256.

**Comment by A-N-Other** [comment]
@spackbot run pipeline

**Comment by tldahlgren** [comment]
Looks like there are CI stack build issues (see https://gitlab.spack.io/spack/spack/-/pipelines/419039).

**Comment by A-N-Other** [comment]
@tldahlgren Looks like we now have liftoff ...

File: var/spack/repos/builtin/packages/star/package.py [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
     """STAR is an ultrafast universal RNA-seq aligner."""
 
     homepage = "https://github.com/alexdobin/STAR"
     url = "https://github.com/alexdobin/STAR/archive/2.7.6a.tar.gz"
 
+    version("2.7.10b", sha256="0d1b71de6c5be1c5d90b32130d2abcd5785a4fc7c1e9bf19cc391947f2dc46e5")
+    version("2.7.10a", sha256="af0df8fdc0e7a539b3ec6665dce9ac55c33598dfbc74d24df9dae7a309b0426a")
     version("2.7.6a", sha256="9320797c604673debea0fe8f2e3762db364915cc59755de1a0d87c8018f97d51")
     version("2.7.0e", sha256="2fc9d9103bd02811904d41e3a3d50e47c7de17cb55c3b4880ea5f39300a9ba0d")
     version("2.7.0d", sha256="7a757478868dc73fe7619bf6ea302dd642bd30e1c8c1fb4acdbe7fa151cf9fd1")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/star/package.py:19]: I know you didn't change this but the sha256 has apparently changed to `b3152b476161c7da324cf2cc1b55c35d3898b4ba0a57ec4cc699f48399b95799`.

**Review by tldahlgren — APPROVED** [review]
Confirmed the version sha256 (except `2.7.0d`), which appears to have changed, and `spack install` succeeds for `2.7.10b` using gcc@10.3.1

File: var/spack/repos/builtin/packages/star/package.py (hunk 1 above)
Comment (tldahlgren) [reply var/spack/repos/builtin/packages/star/package.py:19]: Let me know if you want to fix this in this PR (though it is unrelated to it).

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/star/package.py:19]: Actually, I'm going to merge anyway.

Comment (A-N-Other) [reply var/spack/repos/builtin/packages/star/package.py:19]: Matching for me:
```
# spack checksum star 2.7.10b 2.7.10a
==> Found 2 versions of star:
  
  2.7.10b  https://github.com/alexdobin/STAR/archive/2.7.10b.tar.gz
  2.7.10a  https://github.com/alexdobin/STAR/archive/2.7.10a.tar.gz

==> Fetching https://github.com/alexdobin/STAR/archive/2.7.10b.tar.gz
==> Fetching https://github.com/alexdobin/STAR/archive/2.7.10a.tar.gz

    version("2.7.10b", sha256="0d1b71de6c5be1c5d90b32130d2abcd5785a4fc7c1e9bf19cc391947f2dc46e5")
    version("2.7.10a", sha256="af0df8fdc0e7a539b3ec6665dce9ac55c33598dfbc74d24df9dae7a309b0426a")
``

**Pull request merged**